public/assets/scss/_homepage.scss

```diff
@@ -78,6 +78,12 @@
     display: flex;
     -webkit-display: flex;
     align-items: center;
+    @media (min-width: 992px) {
+      &.col-lg-8 {
+        width: 66% !important;
+      }
+    }
+
```

Comment:
Removed the newline https://github.com/SpongePowered/SpongeHome/pull/97/commits/94184adfa80e96d566a1b0b367af8a48e0f651a4